.github/ISSUE_TEMPLATE/docs.yaml

```diff
@@ -0,0 +1,45 @@
+name: Documentation
+description: Submit a documentation request
+labels: "documentation"
+body:
+  - type: markdown
+    attributes:
+      value: >
+        Thanks for taking the time to fill out this documentation request! Before you submit,
+        please make sure there isn't an existing issue concerning your request. If there
+        is, please join that discussion instead.
+  - type: dropdown
+    id: type
+    attributes:
+      label: Request type
+      description: Are you requesting a fix or an enhancement to the docs?
+      options:
+        - Fix
+        - Enhancement
+    validations:
+      required: true
```

Comment:
I'd like to consider cutting this. It doesn't save us any time in tracking because it's manually parsed. There are many cases where fixes turn out to be enhancements and vice versa. I think making the distinction (if it's helpful – doubtful) will remain with us during triage.